src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -56,7 +56,7 @@ class InputModel::InputModelTFImpl {
     InputModelTFImpl(const GraphIterator::Ptr& graph_iterator,
```

Comment:
Good question. It is declared in "input_model.hpp" as an incomplete class (to make a weak link between entities). And because we don't explicitly use it anywhere (we don't count usage as an pointer) - we don't need to include full declaration.